vllm/config/parallel.py

```diff
@@ -58,6 +59,9 @@ class EPLBConfig:
     This is turned off by default since it will cause communication overhead.
     """
 
+    eplb_policy_type: EPLBPolicyType = "default_eplb"
```

Comment:
We are already inside `EPLBConfig`, we can simplify these names a bit:

```suggestion
    policy: EPLBPolicy = "default"
```